src/context_store/dashboard/websocket_manager.py

```
@@ -0,0 +1,85 @@
+            return
+
+        # Embed channel into payload
+        payload_with_channel = {**payload, "channel": channel}
+
+        async def safe_send(ws: WebSocket) -> None:
+            try:
+                await asyncio.wait_for(ws.send_json(payload_with_channel), timeout=1.0)
+            except asyncio.TimeoutError:
+                logger.warning("WS send timeout, disconnecting client")
+                self.disconnect(ws)
+            except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)